apps/src/templates/studioHomepages/TeacherSections.jsx

```diff
@@ -21,8 +24,32 @@ class TeacherSections extends Component {
 
   render() {
     const {queryStringOpen, locale} = this.props;
+
+    /*
+     * TODO (dmcavoy): Remove for May 31st launch
+     * Temporary notification for the experiment facilitators will be using to
+     * see the mini rubric and assessment re-design from May 1 to May 31st
+     */
+    const inMiniRubricExperiment = experiments.isEnabled(
+      experiments.MINI_RUBRIC_2019
+    );
     return (
       <div id="classroom-sections">
+        {inMiniRubricExperiment && (
+          <Notification
+            type={'bullhorn'}
+            notice={'Experiment Enabled'}
+            details={
+              <UnsafeRenderedMarkdown
+                markdown={
+                  'The Mini Rubric and Assessment Re-design experiment is enabled for your account. To disable this experiment go [here](https://studio.code.org/home?disableExperiments=2019-mini-rubric).'
```

Comment:
nit: this can just be `/home?disableExperiments=2019-mini-rubric`, and that will let it work in all environments.